dbt_project.yml

```
@@ -1,7 +1,7 @@
 name: 'hubspot_source'
-version: '0.2.0'
+version: '0.2.1'
```

We should update the this package to include custom schemas and then upgrade to v0.3.0
```suggestion
version: '0.3.0'
```